mock_storage.go

```diff
@@ -0,0 +1,37 @@
+package pazuzu
+
+import (
+	"regexp"
+
+	"github.com/zalando-incubator/pazuzu/shared"
+)
+
+type TestStorage struct{}
```

Comment:
What do you think about moving mock_* to mock package? and synchronize naming between type and file? (ex mock/Storage.go and Storage and use it in tests importing pazuzu/mock/)